src/cljs/main/broadfcui/common/gcs_file_preview.cljs

```diff
@@ -28,34 +28,40 @@
           (labeled "Google Bucket" (:bucket-name props))
           (labeled "Object" (:object props))
           [:div {:style {:marginTop "1em"}}
-           [:div {} "Previews for some filetypes may be unsupported. "]
+           [:div {} (when-not (or (= data-size "0") (clojure.string/blank? data-size))
```

Comment:
We're checking `(or (= data-size "0") (clojure.string/blank? data-size))` several times, so let's pull this test out into a common function.